gnmi_server/server.go

```diff
@@ -5,16 +5,17 @@ import (
 	"errors"
 	"fmt"
 	"github.com/Azure/sonic-mgmt-common/translib"
-	"github.com/sonic-net/sonic-gnmi/common_utils"
-	spb "github.com/sonic-net/sonic-gnmi/proto"
-	spb_gnoi "github.com/sonic-net/sonic-gnmi/proto/gnoi"
-	spb_jwt_gnoi "github.com/sonic-net/sonic-gnmi/proto/gnoi/jwt"
-	sdc "github.com/sonic-net/sonic-gnmi/sonic_data_client"
+	"github.com/Azure/sonic-mgmt-common/translib/transformer"
```

Comment:
Should not include transformer package here. Only interface between the gNMI server management common packages should be through translib

Reply:
Done